examples/hex_neighbors.rs

```diff
@@ -447,24 +465,18 @@ fn main() {
                 })
                 .set(ImagePlugin::default_nearest()),
         )
-        // Initialize the cursor pos at some far away place. It will get updated
-        // correctly when the cursor moves.
-        .insert_resource(CursorPos(Vec3::new(-100.0, -100.0, 0.0)))
         .add_plugin(TilemapPlugin)
-        .add_startup_system_to_stage(StartupStage::PreStartup, spawn_assets)
-        .add_startup_system_to_stage(StartupStage::Startup, spawn_tilemap)
+        .init_resource::<CursorPos>()
+        .init_resource::<TileHandleHexCol>()
+        .init_resource::<TileHandleHexRow>()
+        .init_resource::<FontHandle>()
+        .add_startup_system(spawn_tilemap)
         .add_startup_system_to_stage(StartupStage::PostStartup, spawn_tile_labels)
         .add_startup_system_to_stage(StartupStage::PostStartup, spawn_map_type_label)
         .add_system_to_stage(CoreStage::First, camera_movement)
         .add_system_to_stage(CoreStage::First, update_cursor_pos.after(camera_movement))
-        .add_system_to_stage(CoreStage::Update, swap_map_type)
-        .add_system_to_stage(
-            CoreStage::Update,
-            hover_highlight_tile_label.after(swap_map_type),
-        )
-        .add_system_to_stage(
-            CoreStage::Update,
-            highlight_neighbor_label.after(hover_highlight_tile_label),
-        )
+        .add_system(swap_map_type)
+        .add_system(hover_highlight_tile_label.after(swap_map_type))
```

Comment:
This is awesome, we should really move away from using stages!

Reply:
I would love to continue cleaning that up later. I'm pretty sure these examples don't need any `add_*_to_stage` to work.

Reply:
Yeah, quite possibly. I think the only time they are definitely needed is when we have commands that need to be flushed before the next stage. I can't remember how much that's true for many of the examples.